libgalois/src/analytics/cdlp/cdlp.cpp

```diff
@@ -32,7 +32,7 @@ namespace {
 /// Set to 10 same as Graphalytics benchmark.
 const unsigned int kMaxIterations = 10;
 
-struct CdlpSynchronousAlgo {
+struct CdlpAlgo {
```

Comment:
I am not sure if this is the right place for this common code. It was supposed to be at a place where all apps can use it. You can create a separate `ConnectedComponentsBase.h` file for this, I think.